css/properties/box-lines.json

```diff
@@ -0,0 +1,67 @@
+{
+  "css": {
+    "properties": {
+      "box-lines": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/box-lines",
+          "support": {
+            "chrome": {
+              "version_added": true,
```

Comment:
Was removed in Chrome flavors, see https://www.chromestatus.com/features/5393405823680512

Reply:
Thanks, fixed in dda2386.

Reply:
Added FF data as well in aca1847 (no support).